paasta_tools/marathon_tools.py

```diff
@@ -495,6 +496,7 @@ def load_service_namespace_config(service, namespace, soa_dir=DEFAULT_SOA_DIR):
     # and there's other things that appear in the smartstack section in
     # several cases.
     key_whitelist = set([
+        'healthcheck_mode',
```

Comment:
It doesn't took like that's necessary because line 515 adds everything in the whitelist to the configuration.